zk_toolbox/crates/config/src/ecosystem.rs

```diff
@@ -117,7 +122,11 @@ impl EcosystemConfig {
             configs: config.configs,
             l1_batch_commit_data_generator_mode: config.l1_batch_commit_data_generator_mode,
             l1_network: self.l1_network,
-            link_to_code: self.link_to_code.clone(),
+            link_to_code: self
```

Comment:
This is necessary because if `link_to_code` is relative, we cannot use it to generate consistent paths.

The error arose in the following piece of code:

https://github.com/matter-labs/zksync-era/blob/ef12716e71de70537fa7bee82cd6360091798668/zk_toolbox/crates/zk_supervisor/src/commands/integration_tests.rs#L33-L44

Given `link_to_code` is `.`, when changing the dir to `CONTRACTS_TEST_DATA_PATH` it uses the current directory as base, which had been changed previously and is no longer the original zksync_era directory.